lms/templates/calculator/toggle_calculator.html

```diff
@@ -4,141 +4,150 @@
 %>
 
 <div class="calc-main">
-  <button title="${_('Open Calculator')}" aria-controls="calculator_wrapper" aria-expanded="false" class="calc utility-control-button">
-    <span class="icon fa fa-calculator" aria-hidden="true"></span>
-    <span class="utility-control-label sr">${_("Open Calculator")}</span>
-  </button>
+    <button title="${_('Open Calculator')}" aria-controls="calculator_wrapper" aria-expanded="false" class="calc utility-control-button">
```

Comment:
It would be best for you to fix the safe template issues with this file while you are editing it. Please take a look at the quality report when it finishes running.